.circleci/config.yml

```diff
@@ -173,6 +173,8 @@ commands:
     steps:
       - restore_cache:
           keys:
+            - go-v16-{{ .Branch }}
+            - go-v16-master
             - go-v15-{{ .Branch }}
```

Comment:
Do we need to keep both `go-v15-{{ .Branch }}` and `go-v16-{{ .Branch }}` ?

Reply:
yes/no - it'll use the first one it finds, the old one will age out

Reply:
ah, got it. It makes sense